projects/storefrontlib/cms-components/misc/icon/icon.model.ts

```diff
@@ -45,8 +45,14 @@ export enum ICON_TYPE {
   OFF = 'OFF',
   LINK_OUT = 'LINK_OUT',
   EXPAND_ARROWS = 'EXPAND_ARROWS',
+  CLIPBOARD_LIST = 'CLIPBOARD_LIST',
```

Comment:
If adding them is breaking change, maybe augment this enum in asm.

Reply:
I don't believe this is a breaking change.  We are only adding new values to the enum, so no previously used values would be changed.  Additionally, enums cannot be extended, so there is little chance of conflicts.